QuizMaven

```diff
@@ -0,0 +1,92 @@
+QUIZ
+Note: You could verify your quiz results online https://www.tutorialspoint.com/maven/maven_online_quiz.htm but please
+be honest and answer to this questions before getting to an answer from the site provided.
+
+1.	What of the following is true about POM?
+
+•	It is fundamental Unit of Work in Maven.
+•	It is an XML file.
+•	Both of the above
+•	None of the above.
+
+ANSWER: Both of the above
+
+2.	Which of the following configuration element is present in POM.xml?
+
+•	developers
+•	mailing list
+•	Both of the above.
+•	None of the above.
+
+ANSWER: Both of the above
+
+3.	Which of the following phase in maven life cycle compiles the source code of the project?
+
+•	validate
+•	compile
+•	test
+•	package
+
+ANSWER: compile
+
+4.	Which of the following is true about Maven repository?
+
+•	A repository is a place i.e. directory where all the project jars, library jar, plugins or any other project
+specific artifacts are stored.
+•	Maven repository is used by Maven.
+•	Both of the above.
+•	None of the above.
+
+ANSWER: Both of the above
+
+5.	Which of the following maven plugin executes during the build and should be configured in the element of pom.xml?
+
+•	Build plugins
+•	Reporting plugins
+•	Both of the above
+•	None of the above.
+
+ANSWER: Build plugins
+
+6.	Which of the following is the default value of packaging?
+
+•	ear
+•	war
+•	jar
+•	pom
+
+ANSWER: jar
+
+7.	Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and
+documentation from a central piece of information.
+•	true
+•	false
+
+ANSWER: true
+
+8.	Which of the following is correct about super POM?
+
+•	Maven use the effective pom (configuration from super pom plus project configuration) to execute relevant goal.
+•	It helps developer to specify minimum configuration detail in his/her pom.xml. Although configurations can be
+overridden easily.
+•	Both of the above.
+•	None of the above.
+
+ANSWER: Both of the above
+
+9.	Which of the following phase in maven life cycle compiles the test source code into the test destination directory?
+
+•	test-compile
+•	process-test-sources
+•	generate-test-sources
+•	None of the above.
+
+ANSWER: test-compile
+
+10.	Which of the following phase in maven life cycle processes and deploys the package if necessary into an environment where integration tests can be run?
+•	process-resources
+•	pre-integration-test
+•	integration-test
+•	None of the above.
+
+ANSWER: integration-test
```

Comment:
`pre-integration-test` | perform actions required before integration tests are executed. This may involve things such as setting up the required environment.

Reply:
https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html
pre-integration-test | perform actions required before integration tests are executed. This may involve things such as setting up the required environment.
-- | --
integration-test | process and deploy the package if necessary into an environment where integration tests can be run.

Reply:
I will answer to you with same argument as you mentioned for `compile` question :)
It is `pre-integration-test`. 
`integration-test` phase kick off `pre-integration-test` first and then run tests